webapp.yml

```diff
@@ -0,0 +1,83 @@
+# Description
+# ===========
+# This playbook creates an Azure Linux Web app with Java container Tomcat 8.5,
+# then build a Java hello world app, ftp the war file to the Web app.
+# Please note, it's recommended to use CI/CD tool to do webapp deployment, such as Jenkins.
```

Comment:
Please note that.... - sounds better
maybe "to use CI/CD tool (such as Jenkins)" would be better instead of attaching ", such as Jenkins." at the end